File: src/main/java/insogroup/utils/exceptions/ClassExtractException.java
package insogroup.utils.exceptions;

import java.io.IOException;

/**
 * Throws when extraction class was failed
 */
public class ClassExtractException extends IOException {
    public ClassExtractException(String message) {
        super(message);
    }

    public ClassExtractException(String message, Throwable cause) {
        super(message, cause);
    }

    public ClassExtractException(Throwable cause) {
        super(cause);
    }

    public ClassExtractException() {
    }
}

File: src/main/kotlin/com/insogroup/utils/ClassExtractor/exceptions/ClassExtractException.kt
package com.insogroup.utils.ClassExtractor.exceptions

import java.io.IOException

/**
 * Throws when extraction class was failed
 */
class ClassExtractException : IOException {
    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}
}
